osu.Game/Screens/Play/HUD/ArgonHealthDisplay.cs

```diff
@@ -228,53 +216,22 @@ private void finishMissDisplay()
 
             if (Current.Value > 0)
             {
-                // TODO: REMOVE THIS. It's recreating textures.
-                glowBar.TransformTo(nameof(BarPath.BarColour), main_bar_colour, 300, Easing.In);
-                glowBar.TransformTo(nameof(BarPath.GlowColour), main_bar_glow_colour, 300, Easing.In);
+                glowBar.TransformTo(nameof(ArgonHealthDisplayBar.BarColour), main_bar_colour, 300, Easing.In);
```

Comment:
hmm, i was thinking this part could also be moved to the shader code (basically setting the required colours and then providing an easing value between them, since there's only two), but maybe we can leave that for a follow-up step in case we need more flexibility in the future.